packages/payment-detection/src/types.ts

```diff
@@ -15,18 +15,46 @@ export interface ContractBasedDetector {
 }
 
 /** Generic info retriever interface */
+
+/** Params for TheGraph-based getTransferEvents */
+export type TransferEventsParams = {
```

Comment:
I moved the type here since things are getting pretty consistent between EVM and Near chains.